jobs/uaa/spec

```diff
@@ -1335,3 +1335,58 @@ properties:
       and the system default identity zone policy will apply to all zones. When set
       to false, the CORS policy for custom identity zones will be honored.
     default: true
+
+  # Global references
+  uaa.global.jwt:
```

Comment:
maybe rename this to jwk, e.g. uaa.global.jwk

This entry is mainly for configuration of keys for other sections, e.g. now we want use it it jwtClientAuthentication
but later we also want remove the keys from TokenPolicy entry in a uaa default zone

OIDC IdP config in uaa.yml would look like

      oidc.proxy:
        type: oidc1.0
        passwordGrantEnabled: true
        discoveryUrl: https://accounts.ondemand.com/.well-known/openid-configuration
        issuer: https://accounts.ondemand.com
        scopes:
          - openid
          - email
          - profile
        linkText: Login with SAP
        relyingPartyId: 0000000
        groupMappingMode: AS_SCOPES
        jwtClientAuthentication:
          alg: RS512
          key: ${global.jwk.oidc-trust.key}
          cert: ${global.jwk.oidc-trust.cert}
        attributeMappings:
          user_name: email
          external_groups:
            roles
        showLinkText: true

Reply:
@torsten-sap can you please comment if you want stay with jwt or jwk? 

We want use the section to store keys in yaml therefore I think jwk ( json web key ) would be more precise

Reply:
"uaa.global.jwk" is fine for me.

Reply:
done